FirebaseDynamicLinks/Tests/Unit/FIRDynamicLinksTest.m

```diff
@@ -491,7 +491,7 @@ - (void)testUniversalLink_NoDeepLink {
 //  https://a.firebase.com/mypath
 - (void)testDynamicLinkFromUniversalLinkURLWithCustomDomainLink {
   self.service = [[FIRDynamicLinks alloc] init];
-  NSString *durableDeepLinkString = @"https://a.firebase.com/mypath/?link=abcd";
+  NSString *durableDeepLinkString = @"https://a.firebase.com/mypath/?link=http://abcd";
```

Comment:
Does this mean there's a change in functionality?

Reply:
The Client SDK will make sure the link query parameter is having and http/https link to consider it as a valid link which was not done before. So we won't make a backend call to validate the URL. Previously, for any value in link, we were making backend calls considering it as a valid DL.  This is in sync with Server side validations.

Reply:
So to confirm, previously the SDK would send off a request to the backend but the backend would reject it, now we're doing that rejection in the SDK?

Reply:
Yes